spec/system/item_system_spec.rb

```diff
@@ -167,6 +167,24 @@
       expect(expanded_row).to have_content num_tampons_in_donation
       expect(expanded_row).to have_content num_tampons_second_donation
     end
+
+    it "should highlight total quantity if it is below minimum quantity" do
```

Comment:
Can this be moved to a request spec? If there's no real interactivity then there isn't a reason to make it a (more heavier, flakier) system spec.

Reply:
Okay, looking into it.